src/nvim/highlight_group.c

```diff
@@ -0,0 +1,2792 @@
+
```

Comment:
I don't think that comment is particularly useful, especially since we have a separate `highlight.c`.

I've added this instead:
```
// highlight_group.c: code for managing highlight groups
```